specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-02-01/managedClusters.json

```diff
@@ -3687,6 +3687,11 @@
           "title": "The version of Kubernetes the Managed Cluster is running.",
           "description": "When you upgrade a supported AKS cluster, Kubernetes minor versions cannot be skipped. All upgrades must be performed sequentially by major version number. For example, upgrades between 1.14.x -> 1.15.x or 1.15.x -> 1.16.x are allowed, however 1.14.x -> 1.16.x is not allowed. See [upgrading an AKS cluster](https://docs.microsoft.com/azure/aks/upgrade-cluster) for more details."
```

Comment:
Shouldn't this also have a section saying that now you can specify only `1.20` and get `1.20.13`?